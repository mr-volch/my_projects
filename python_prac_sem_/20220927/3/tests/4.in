90,60
70,80
1,0
0,1

