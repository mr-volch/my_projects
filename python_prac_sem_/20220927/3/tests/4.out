90,60
70,80
